tests/probes/textfilecontent54/test_filecontent_non_utf.sh

```diff
@@ -14,7 +14,16 @@ function test_filecontent_non_utf {
     cp ${srcdir}/test_filecontent_non_utf.${FILE_SUFFIX} ${test_dir}/${FILE_SUFFIX}
 
     cp ${srcdir}/test_filecontent_non_utf.oval.xml ${test_dir}/oval.xml
-    sed -i "s:TEST_FILE:${test_dir}/${FILE_SUFFIX}:" ${test_dir}/oval.xml
+
+    case $(uname) in
```

Comment:
How about adding `xsed` function to the `test_common.sh` and using `sed` or `gsed` there based on `uname`?